src/utils/mod.rs

```diff
@@ -610,6 +611,73 @@ pub enum PublicIdUnion {
     Ecc(Box<TPMS_ECC_POINT>),
 }
 
+// The digests field type is from bindgen which does not implement Debug on them.
```

Comment:
I'm confused by this - isn't `Digest` the structure defined below on line 1745, as that seems to have `Debug`?

Reply:
Yep, it sure is. This was from an earlier version of the code where it had `Vec<TPM2B_DIGEST>` as `digests`.